pkg/resource/validations/kmsArnRegexValidation_test.go

```diff
@@ -0,0 +1,48 @@
+package validations
+
+import (
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Validations", func() {
+	Describe("validateKMSKeyARN", func() {
+		var (
+			kmsKeyARN  string
+		)
+
+		BeforeEach(func() {
+			kmsKeyARN = ""
+		})
+
+		Context("when kmsKeyARN is nil", func() {
```

Comment:
@den-rgb Does this test pass? From the code I don't see you returning an `error` if the `kmsKeyArn` is `nil`?

Reply:
It expects an error on line 22, it gets that error from line 21

Reply:
That test was checking if it was empty not nil, I'll add an extra test case for checking if it's nil and I'll refactor the context